inst/chunks/fun_com_poisson.stan

```diff
@@ -1,118 +1,139 @@
-  // log approximate normalizing constant of the COM poisson distribuion
-  // approximation based on doi:10.1007/s10463-017-0629-6
-  // Args: see log_Z_com_poisson()
-  real log_Z_com_poisson_approx(real log_mu, real nu) {
-    real nu_mu = nu * exp(log_mu);
-    real nu2 = nu^2;
-    // first 4 terms of the residual series
-    real log_sum_resid = log1p(
-      nu_mu^(-1) * (nu2 - 1) / 24 +
-      nu_mu^(-2) * (nu2 - 1) / 1152 * (nu2 + 23) +
-      nu_mu^(-3) * (nu2 - 1) / 414720 * (5 * nu2^2 - 298 * nu2 + 11237)
-    );
-    return nu_mu + log_sum_resid  -
-      ((log(2 * pi()) + log_mu) * (nu - 1) / 2 + log(nu) / 2);
+// log approximate normalizing constant of the COM poisson distribuion
+// based on equations (4) and (31) of doi:10.1007/s10463-017-0629-6
+// Args: see log_Z_com_poisson()
+real log_Z_com_poisson_approx(real log_mu, real nu) {
+  real nu2 = nu^2;
+  real log_common = log(nu) + log_mu/nu;
+  array[4] real resids;
+  real ans;
+  real lcte = (nu * exp(log_mu/nu)) -
+    ( (nu-1)/(2*nu)* log_mu + (nu-1)/2*log(2*pi()) + 0.5 *log(nu));
+  real c_1 = (nu2-1)/24;
```

Comment:
can you please add proper spacing everywhere?

Reply:
Done c560adcaa76a2d846dd0703decfcdfa2042f7a99